### DISCUSSION TITLE
Notifications on rescheduling post

### DISCUSSION THREAD

**Discussion by jorr** [publishpress/PublishPress-Planner#1278] [Feature Requests]
I suggest a new kind of notifications that trigger when a post is rescheduled, either via calendar, quick edit or edit.

Why: in real life calendar scheduling mght need to be revised and flexible. If I reschedule an article that the author notified me they will not be able to accomplish, and the requires shuffling other articles around, too, then other parties involved should be notified. Granted, they will get reminders related to the new scheduled date but this is even more confusing, especially if their new date is sooner than the original one. It would be great if PP could notify on rescheduling. Since there is no status change in this case, the status change kind of notification doesn't serve this purpose.

I further suggest that this notification triggers at least 10 minutes (ideally, configurable) after rescheduling and act cumulative (the usual case is that editors move around posts in the calendar until they are happy. it would be notification hell if each move triggers a notification. but after a set amount of time we can just notify about the latest valid rescheduling).

**Comment by andergmartins** [comment]
Related to #495
